sample/bin/template.js

```diff
@@ -0,0 +1,50 @@
+const { MainError } = require('@effection/node');
+const merge = require('lodash.merge');
+const chalk = require('chalk');
+
+const { formatErr } = require('./console-helpers');
+const { baseTemplate, bigTestTemplate, cypressTemplate, jestTemplate } = require('./templates');
+
+function processTemplate() {
+  let pkgjson;
+  let files;
+  let templateName;
+  if(process.argv.includes('--only')){
+    let onlyFlagIndex = process.argv.indexOf('--only');
+    templateName = process.argv[onlyFlagIndex + 1];
+    if(templateName !== 'cypress' && templateName !== 'bigtest' && templateName !== 'jest'){
+      throw new MainError({ message: `${formatErr(`The --only flag received an unrecognized framework.`)}` });
```

Comment:
In order to make the error message helpful, we should provide what the the expected value and the actually value was. E.g.

```
expected the value of `--only` to be one of `jest`, `cypress`, or `bigtest`, but was `bligga-blorg`
```

Reply:
😆 perhaps. Is there a way to dial it down just a tad without losing the magic?

Reply:
![Screen Shot 2021-03-19 at 9 38 08 AM](https://user-images.githubusercontent.com/29791650/111788815-e269f980-8896-11eb-926b-f0b65b521b76.png)

We could just keep it all the same color